src/http.rs

```diff
@@ -52,6 +52,7 @@ impl HttpError {
 }
 
 pub type HttpClientFuture = <HttpClient as Service<http::Request<Body>>>::Future;
+pub type BodyBox = http_body::combinators::BoxBody<hyper::body::Bytes, hyper::Error>;
```

Comment:
I also write this type alias in multiple files, how can I prevent rewriting an alias or is it fine for the purposes of this PR? I was required to change the Body type to BodyBox by the rust compiler to get the changes to compile

Reply:
I will add the change to use the pub type BodyBox from the `src/http.rs` file